packages/rspeedy/core/src/plugins/dev.plugin.ts

```diff
@@ -83,12 +85,19 @@ export function pluginDev(
       })
 
       const require = createRequire(import.meta.url)
+      const __dirname = path.dirname(fileURLToPath(import.meta.url))
 
       api.modifyBundlerChain((chain, { isProd }) => {
         if (isProd) {
           return
         }
         const rsbuildPath = require.resolve('@rsbuild/core')
+        const rspeedyDir = path.dirname(
+          require.resolve('@lynx-js/rspeedy/package.json', {
+            paths: [__dirname],
```

Comment:
Why do we need this?

I think `require.resolve` starts from current directory by default